src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -0,0 +1,49 @@
+package codeu.controller;
```

Comment:
This file is related to Quin's work, right?  I assume this is left over from a merge?  If so, you should remove it.